cmd/root.go

```diff
@@ -144,10 +152,17 @@ func getImageForName(imageName string) (pkgutil.Image, error) {
 			return pkgutil.Image{}, err
 		}
 
-		img, err = daemon.Image(ref, &daemon.ReadOptions{})
+		start := time.Now()
+		img, err = daemon.Image(ref, &daemon.ReadOptions{
+			Buffer: true,
+		})
 		if err != nil {
 			return pkgutil.Image{}, err
 		}
+		elapsed := time.Now().Sub(start)
+		logrus.Infof("retrieving image from daemon took %f seconds", elapsed.Seconds())
+		// TODO(nkubala): remove this when we can set compression level in containerregistry
+		noCache = true // force noCache if image is in daemon
```

Comment:
How does this work if one image is in the registry and one isn't?

Reply:
ah yeah, good point. changed to use a different variable